tools/distpartitioning/parmetis_preprocess.py

```diff
@@ -121,10 +121,14 @@ def process_and_write_back(data_df, idx):
         for idx in file_idxes[rank]:
             reader_fmt_meta = {
                 "name": etype_info[constants.STR_FORMAT][constants.STR_NAME],
-                "delimiter": etype_info[constants.STR_FORMAT][
-                    constants.STR_FORMAT_DELIMITER
-                ],
             }
+            if (
+                etype_info[constants.STR_FORMAT][constants.STR_NAME]
+                == constants.STR_CSV
+            ):
+                reader_fmt_meta["delimiter"] = etype_info[constants.STR_FORMAT][
+                    constants.STR_FORMAT_DELIMITER
+                ]
```

Comment:
`if ():` is introduced by the lintrunner.

Reply:
It's introduced by the lintrunner because the condition is too long and has to be broken up into multiple lines. If you use a shorter condition that will not be an issue.

Reply:
Changed the code as suggested.